portfolio/src/main/webapp/styles/style.css

```diff
@@ -39,18 +40,30 @@ img {
 
 /* Left flex element of 'outer-nav' container */
 #home-logo {
-  color: #55FBDC;
-  flex-basis: 12%;
-  flex-shrink: 0;
-  font-size: 22pt;
+  background:rgb(40,40,60);
```

Comment:
nit: please add space after ":" here and the rest of the file. thx

Reply:
See #12 .